modules/stochastic_tools/doc/content/source/samplers/QuadratureSampler.md

```diff
@@ -10,7 +10,65 @@ Numerical quadrature is a method of selecting specific points and weights in int
+where $y(x)$ is the function being integrated, $f(x)$ is weighting function, $N_q$ is the number of quadrature points, $w_q$ and $x_q$ are quadrature weights and points, respectively. The sampler generates the points and weights based on the weighting function and the desired integration order. A uniform weighting function uses Gauss-Legendre quadrature and a normal weighting function uses Gauss-Hermite quadrature. These two quadratures can exactly integrate a polynomial of order $2N_q-1$.
+
+For multidimensional quadratures, the following options are available by setting [!param](/Samplers/QuadratureSampler/sparse_grid):
+
+- `none` (default): Tensor grid
+- `smolyak`: Smolyak sparse grid
+- `clenshaw-curtis`: Clenshaw-Curtis sparse grid
+
+### Tensor Grid
+
+A tensor grid is created created by taking a Cartesian product of the points and a Kroenecker product of the weights from each dimensions' full order quadrature set. The resulting number of points is $N_q^D$, where $D$ is the number of dimensions. [tensor-npts] shows the number of points several different $D$ and $N_q$. [tensor-grid] shows the points of a tensor grid of Gauss-Legendre quadrature with $D=2$ and $N_q=7$.
+
+!table id=tensor-npts caption=Resulting number of grid points for tensor grid
+| $N_q$ | $D=2$ | $D=5$ | $D=8$ |
+| - | - | - | - |
+| 2 | 4 | 32 | 256 |
+| 3 | 9 | 243 | 6,561 |
+| 5 | 25 | 3,125 | 390,625 |
+| 7 | 49 | 16,807 | 5,764,801 |
+
+!media media/tensor_grid.svg id=tensor-grid
+
+### Smolyak Sparse Grid
+
+A Smolyak sparse grid is a multidimensional quadrature meant to integrate a complete monomial. This type of grid is effective for use with [PolynomialChaos.md]. The idea is that instead of taking the cartesian product of full order quadratures, it uses a combination of lower order quadratures to complete the monomial space. The set for a tensor grid can be described as:
+
+!equation
+\mathcal{A}(D,N_q) = Q_1^{N_q} \otimes\dots\otimes Q_D^{N_q} ,
+
+where $D$ is the number of dimensions and $Q_d^{j}$ is the one-dimensiona quadrature set for dimension $d$ with $j$ points. In contrast, the Smolyak grid is described as:
+
+!equation
+\mathcal{A}(D,N_q) = \sum_{N_q-D \leq |\bm{i}| \leq N_q-1} (-1)^{N_q+D-|\bm{i}|-1} \binom{N_q+D-1}{N_q+D-|\bm{i}|-1} \left(Q_1^{i_1} \otimes\dots\otimes Q_D^{i_D}\right) ,
+
+where $|\bm{i}| = \sum_{d=1}^D i_d$. The Smolyak sparse grid scheme can reduce the number of points significantly in high dimensional space. [smolyak-npts] shows the number of points several different $D$ and $N_q$. [smolyak-grid] shows the points of a Smolyak grid of Gauss-Legendre quadrature with $D=2$ and $N_q=7$. See [!cite](gerstner1998numerical) for more details regarding sparse grids.
```

Comment:
Does this cover all the methods? If so, can you move this reference to the top where you list the three available.

Reply:
Actually, yes it does. I will reorganize. Good catch.